<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <item>
         <property name="text">
          <string>General</string>
         </property>
         <property name="icon">
          <iconset theme="preferences-system">
           <normaloff/>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>netctl</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resources.qrc">
           <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sudo</string>
         </property>
         <property name="icon">
          <iconset theme="gksu">
           <normaloff/>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>wpa_supplicant</string>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resources.qrc">
           <normaloff>:/wifi.png</normaloff>:/wifi.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Other</string>
         </property>
         <property name="icon">
          <iconset theme="configure">
           <normaloff/>
          </iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_general">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="layout_language">
            <item>
             <widget class="QLabel" name="label_language">
              <property name="text">
               <string>Language</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_language">
              <property name="toolTip">
               <string>Select a language</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_info">
            <property name="text">
             <string>You will need to restart the application</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer_general">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>424</width>
              <height>223</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_netctl">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="layout_systemctlPath">
            <item>
             <widget class="QLabel" name="label_systemctlPath">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>systemctl path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_systemctlPath">
              <property name="toolTip">
               <string>Path to systemctl</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_systemctlPath">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_netctlPath">
            <item>
             <widget class="QLabel" name="label_netctlPath">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>netctl path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_netctlPath">
              <property name="toolTip">
               <string>Path to netctl</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_netctlPath">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_netctlAutoPath">
            <item>
             <widget class="QLabel" name="label_netctlAutoPath">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>netctl-auto path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_netctlAutoPath">
              <property name="toolTip">
               <string>Path to netctl-auto</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_netctlAutoPath">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_netctlAutoService">
            <item>
             <widget class="QLabel" name="label_netctlAutoService">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>netctl-auto service</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_netctlAutoService">
              <property name="toolTip">
               <string>Name of netctl-auto systemd service</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_profilePath">
            <item>
             <widget class="QLabel" name="label_profilePath">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Profile path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_profilePath">
              <property name="toolTip">
               <string>Path to profile directory</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_profilePath">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="spacer_netctl">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>424</width>
              <height>122</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_sudo">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="layout_sudo">
            <item>
             <widget class="QLabel" name="label_sudo">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>sudo path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_sudo">
              <property name="toolTip">
               <string>Path to sudo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_sudo">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="spacer_sudo">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>424</width>
              <height>244</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_wpaSup">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="layout_wpaSupPath">
            <item>
             <widget class="QLabel" name="label_wpaSupPath">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>wpa_supplicant path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_wpaSupPath">
              <property name="toolTip">
               <string>Path to wpa_supplicant</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_wpaSupPath">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_wpaCliPath">
            <item>
             <widget class="QLabel" name="label_wpaCliPath">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>wpa_cli path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_wpaCliPath">
              <property name="toolTip">
               <string>Path to wpa_cli</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_wpaCliPath">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_wpaActiondPath">
            <item>
             <widget class="QLabel" name="label_wpaActiondPath">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>wpa_actiond path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_wpaActiondPath">
              <property name="toolTip">
               <string>Path to wpa_actiond</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_wpaActiondPath">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_pid">
            <item>
             <widget class="QLabel" name="label_pid">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>PID file</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_pid">
              <property name="toolTip">
               <string>wpa_supplicant PID file</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_wpaSupDrivers">
            <item>
             <widget class="QLabel" name="label_wpaSupDrivers">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>wpa_supplicant drivers</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_wpaSupDrivers">
              <property name="toolTip">
               <string>wpa_supplicant drivers comma separated</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_wpaDir">
            <item>
             <widget class="QLabel" name="label_wpaDir">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>ctrl_interface directory</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_wpaDir">
              <property name="toolTip">
               <string>Path to control directory</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_wpaGroup">
            <item>
             <widget class="QLabel" name="label_wpaGroup">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>ctrl_interface group</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_wpaGroup">
              <property name="toolTip">
               <string>Group of control directory</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="spacer_wpaSup">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>66</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_other">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="layout_interfacesDir">
            <item>
             <widget class="QLabel" name="label_interfacesDir">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Path to interface list</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_interfacesDir">
              <property name="toolTip">
               <string>Path to directory which contains network devices</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_interfaceDir">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_rfkill">
            <item>
             <widget class="QLabel" name="label_rfkill">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Path to rfkill device list</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_rfkill">
              <property name="toolTip">
               <string>Path to directory which contains rfkill devices</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_rfkill">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_interface">
            <item>
             <widget class="QLabel" name="label_interface">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Prefered wireless interface</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_interface">
              <property name="toolTip">
               <string>Prefered wireless interface</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="spacer_other">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>424</width>
              <height>184</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
